packages/commonwealth/.eslintrc.cjs

```diff
@@ -88,27 +92,39 @@ module.exports = {
     '@tanstack/query/exhaustive-deps': 'error',
     '@tanstack/query/prefer-query-object-syntax': 'error',
     'react/destructuring-assignment': [1, 'always'],
-    'react/function-component-definition': [1, { "namedComponents": "arrow-function" }],
-    'react/no-multi-comp': [1, { "ignoreStateless": false }],
-    'react/jsx-curly-brace-presence': [1, { props: "never", children: "never" }],
+    'react/function-component-definition': [
```

Comment:
Go clarify, is this just formatting?

Reply:
Other than the change to `parserOptions`, this is all just formatting - it looks like it's following `prettier` defaults.